metricbeat/mb/testing/data/data_test.go

```diff
@@ -51,12 +51,14 @@ const (
 var (
 	// Use `go test -generate` to update files.
 	generateFlag = flag.Bool("generate", false, "Write golden files")
+	moduleFlag   = flag.String("module", "", "Write golden files")
```

Comment:
Can you change the description?

Reply:
Sorry for this. Changed in the next commit